README.md

```diff
@@ -28,12 +28,81 @@ This repository houses the following packages:
 
 Or, in graph form [^fn1]:
 
-![Dependency graph](assets/dependency-graph.png)
```

Comment:
Looks like the old file is still there. Presumably we can delete that

Reply:
Yup, good call.